examples/test_vault.ipynb

```diff
@@ -0,0 +1,266 @@
+{
```

Comment:
<p>I guess you added the <code>.doc()</code>  call so the function's docstring will be printed:</p><ol><li>If you'll change the function to be part of the notebook this is not needed</li><li>If you keep it that way, please upload the notebook including the outputs, some users don't really run it, just go over it and want to know what's going on</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/500/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Yes, it's here to show the docstring. I'll decide whether to move the method here or not (and keep the output).</p>